linodecli/overrides.py

```diff
@@ -37,3 +41,65 @@ def handle_domains_zone_file(operation, output_handler, json_data) -> bool:
     """
     print("\n".join(json_data["zone_file"]))
     return False
+
+
+@output_override("linodes", "types", OutputMode.table)
+def handle_types_region_prices_list(
+    operation, output_handler, json_data
+) -> bool:
+    # pylint: disable=unused-argument
+    """
+    Override the output of 'linode-cli linodes types' to display regional pricing.
+    """
+    if len(json_data["data"]) < 1:
+        return True
+
+    output = Table()
+    headers = sorted(json_data["data"][0].keys() - ["addons"], key=len)
+    region_price_sub_headers = ["id", "hourly", "monthly"]
+
+    for header in headers:
+        output.add_column(header, justify="center")
+
+    for linode in json_data["data"]:
+        row = []
+        for h in headers:
+            if h == "region_prices":
+                sub_table = Table()
+                for header in region_price_sub_headers:
+                    sub_table.add_column(header, justify="center")
+                for region_price in linode[h]:
+                    region_price_row = []
+                    for header in region_price_sub_headers:
+                        region_price_row += (
+                            Align(str(region_price[header]), align="left"),
+                        )
+                    sub_table.add_row(*region_price_row)
+                row += [sub_table]
+
+            elif h == "price":
+                sub_table = Table()
+                for header in ["hourly", "monthly"]:
+                    sub_table.add_column(header, justify="center")
+                sub_table.add_row(
+                    *[
+                        Align(str(linode[h]["hourly"]), align="left"),
+                        Align(str(linode[h]["monthly"]), align="left"),
+                    ]
+                )
+                row += [sub_table]
```

Comment:
Given there will always only be one entry for `price.hourly` and `price.monthly`, maybe they could become their own columns :eyes:

Reply:
Sounds good. I put them into separate columns.